integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -69,6 +69,12 @@ spec:
 {{ toYaml .Values.master.nodeSelector | trim | indent 8  }}
       {{- else if .Values.nodeSelector }}
 {{ toYaml .Values.nodeSelector | trim | indent 8  }}
+      {{- end }}
+      tolerations:
+      {{- if .Values.master.tolerations }}
+{{ toYaml .Values.master.tolerations | trim | indent 8  }}
+      {{- else if .Values.tolerations }}
```

Comment:
How do you like `.Values.master.tolerations` + `.Values.tolerations`? In other words I think this should be a union instead of if-else.

Reply:
It would be good if `.Values.tolerations` is there which can add tolerations to all the Alluxio components rather than specifying them individually. And if someone wants to run only some components on dedicated nodes, then they can use `.Values.XYZ.tolerations`

Reply:
Yeah that makes sense to me, I can change it to a union of the tolerations.